services/app-web/src/pages/QuestionResponse/UploadQuestions/UploadQuestions.test.tsx

```diff
@@ -414,11 +463,18 @@ describe('UploadQuestions', () => {
                                     user: mockUser(),
                                     statusCode: 200,
                                 }),
-                                fetchStateHealthPlanPackageWithQuestionsMockSuccess(
-                                    {
+                                fetchContractWithQuestionsMockSuccess({
+                                    contract: {
+                                        ...contract,
                                         id: '15',
-                                    }
-                                ),
+                                    },
+                                }),
+                                fetchContractWithQuestionsMockSuccess({
```

Comment:
how come we are calling this twice?

Reply:
Wasn't able to reproduce this in the UI but in the test this needed to be called twice when the mocked submission with questions was used. I updated the test to use the mocked submission to avoid the duplicated call

Reply:
This is due to the `SubmissionSideNav` component fetching as well as the outlet components like Q&A pages and summary pages. I think we will need a refactor on the `SubmissionSideNav` component and how we fetch on those outlet components.